db/migrate/20161118210435_add_question_number.rb

```diff
@@ -0,0 +1,6 @@
+class AddQuestionNumber < ActiveRecord::Migration
+  def change
+    add_column :questions, :number, :integer, unique: true
```

Comment:
We have this in ActiveRecord models from question `has_one :survey` but not in the tables. I think we should keep the `survey_id` on the `question_orders` table because question orders is what connects surveys to questions. We could duplicate the column across two tables but I think it's better to add a more sophisticated validation method on the survey ActiveRecord model. Again, this is more of a nice to have than necessity so I'd be willing to merge without this.